src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1373,41 +1374,66 @@ private AuthenticationResult acquireTokenAfterValidation(CallbackHandler callbac
                     Logger.v(TAG, "Initial request to authenticator");
                     // Log the initial request but not force a prompt
                 }
+   
+                // Launch broker activity
+                // if cache and refresh request is not handled.
+                // Initial request to authenticator needs to launch activity to
+                // record calling uid for the account. This happens for Prompt auto
+                // or always behavior.
+                Logger.v(TAG, "Token is not returned from backgroud call");
```

Comment:
@heidijinxujia Sorry, I really don't understand what's going on here. I don't understand the purpose for the code you added from line 1378 - 1397